registry/tracecat_registry/base/core/contracts.py

```diff
@@ -0,0 +1,48 @@
+from typing import Annotated, Any
+
+from typing_extensions import Doc
+
+from tracecat_registry import registry
+
+
+@registry.register(
+    default_title="Require Condition",
+    description="Check a condition. Raises `AssertionError` if the condition is false. Returns `True` if the condition is true.",
+    namespace="core",
+)
+def require(
+    lhs: Annotated[Any, Doc("Left hand side of the condition.")],
+    rhs: Annotated[Any, Doc("Right hand side of the condition.")],
+    condition: Annotated[
+        str,
+        Doc(
+            "Condition to assert (`==`, `!=`, `>`, `>=`, `<=`, `in`, `not in`, `is`, `is not`)."
+        ),
+    ],
+) -> bool:
```

Comment:
```suggestion
) -> None:
```

Reply:
Since people might try using this, let's keep returning `True` in the happy path to distinguish between `None` that would be returned in the error path.